src/main/java/duke/command/Parser.java

```diff
@@ -0,0 +1,49 @@
+package duke.command;
+
+/**
+ * Class to parse the string command
+ * into the Command class.
+ * @author Oskar Lew
+ * @version 0.1
+ * @since 0.1
+ */
+public class Parser {
+
+    /**
+     * Constructor of Parser.
+     */
+    public Parser() {
+    }
+
+    /**
+     * Method that returns the right Command class
+     * depending on the string input from the user.
+     * @param echo Command from the user.
+     * @return Command class
+     */
+    public Command parse(String echo) {
+        String[] command = echo.split(" ");
+        switch (command[0]) {
```

Comment:
I like how you used the switch statement to make all commands clear and concise :)